### PR TITLE
Animation for Dijkstra&Two pointers&Kruskal

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
wait till my maintenance pr merges
then you can use the code from that instead for consistency

File: content/4_Gold/Shortest_Paths.mdx
Comment (SansPapyrus683) [inline content/4_Gold/Shortest_Paths.mdx:285]: also this is just in the cppsection 💀

**Comment by brebenelmihnea** [comment]
Ok

**Comment by brebenelmihnea** [comment]
@SansPapyrus683 can you merge it now ?

**Comment by SansPapyrus683** [comment]
did you change the code in the animation

**Comment by brebenelmihnea** [comment]
Oh, I thought i was good.  I will change it

**Comment by SansPapyrus683** [comment]
did you? it still seems the same to me

**Comment by brebenelmihnea** [comment]
The code in the animation:
image
The code in module:
image

**Comment by SansPapyrus683** [comment]
did you pull

**Comment by brebenelmihnea** [comment]
?

**Comment by brebenelmihnea** [comment]
I added the kruskal animation too

**Comment by SansPapyrus683** [comment]
in the future when you add additional changes
just make a new pr for them

**Comment by brebenelmihnea** [comment]
Ah ok

**Comment by brebenelmihnea** [comment]
@SansPapyrus683

**Review by SansPapyrus683 — APPROVED** [review]
get another CA to review and we chillin

**Comment by brebenelmihnea** [comment]
Ok. @ryanchou-dev

**Review by ryanchou-dev — COMMENTED** [review]
for the two pointers, it's probably be better if you keep a maximum answer so that you answer the problem.
(right now it seems like you are just moving the pointers with no apparent pattern)

dijkstra's looks good, but i think you should move the edge weights out of the edges to make it easier to read.

**Comment by brebenelmihnea** [comment]
Ok, i will make the changes

**Comment by brebenelmihnea** [comment]
@ryanchou-dev I made the changes

**Review by ryanchou-dev — APPROVED** [review]

**Comment by brebenelmihnea** [comment]
@SansPapyrus683 can you merge it now ?

**Comment by SansPapyrus683** [comment]
not my call bro
wait for ryan's approval

**Comment by SansPapyrus683** [comment]
OH WAIT I THOUGHT HE DIDN'T APPROVE YET LMAOOOO

**Comment by ryanchou-dev** [comment]
LOL

**Pull request merged**
